### PR TITLE
Dapr Graduation Proposal

### PR DESCRIPTION
**Pull request by msfussell**
On behalf of the Dapr Steering Committee, we propose to move the Dapr project to graduation stage.

### Dapr and the cloud-native ecosystem

Dapr is a vendor-neutral, cloud agnostic and API-driven framework that enables developers to solve hard distributed systems challenges like event-driven messaging, state management at scale, distributed locks,  long running workflow apps and many others. Underpinning all of Dapr's APIs are 3 pillars of security, observability and reliability, each one provided out-of-the-box and made configurable.

Dapr integrates with over 100 cloud-based and open-source systems, and uses CNCF projects like OpenTelemetry, gRPC, and SPIFFE at its core.

### Graduation maturity

Dapr has grown tremendously since its inception in late 2019, being a relatively new project in the cloud-native ecosystem. Today, Dapr is the [11th largest](https://docs.google.com/spreadsheets/d/1HnYZriD2uBd-phEp3h3TsRTv1kdk-C7RfUiat-p1pt0/edit#gid=976519966) project in CNCF (of 172 projects), with nearly [3,000 individual contributors](https://dapr.devstats.cncf.io/d/18/overall-project-statistics-table?orgId=1) from over [450 companies](https://dapr.devstats.cncf.io/d/5/companies-table?orgId=1).

Our [Discord community](https://discord.com/invite/ptHhX6jc34) has nearly 6,500 members and the project has [22,300 GitHub stars](https://github.com/dapr/dapr) to date.

Dapr's end users include IBM, HDFC Bank, NASA, Rakuten, Bosch, Fujitsu, Zscaler, Zeiss, Porsche, Vonage (Ericsson), Sharper Image and many others. In addition, Dapr is powering core infrastructure in hyper-scale clouds like Alibaba Cloud and Microsoft Azure.

Based on [analysis of the 2022 CNCF project usage survey](https://blog.palark.com/cncf-cloud-native-projects-usage-stats-2022/) data, Dapr accounts for 10% of production users.

The project [concluded a 2023 security audit](https://www.cncf.io/blog/2023/09/06/dapr-completes-2023-security-audit-increasing-enterprise-confidence/) with 0 critical and 0 highs, prompting the auditors to conclude that "Our overall assessment of Dapr is highly positive. Dapr follows security best practices in both design and implementation. Dapr performed well in this audit demonstrating a strong security posture."

Dapr has a vendor-neutral steering committee with 7 members from 4 companies (Intel, Microsoft, Alibaba, Diagrid) with a fully documented governance process.

### CODE REVIEW

File: proposals/graduation/dapr.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+The project has been [accepted as a CNCF Incubating project](https://www.cncf.io/blog/2021/11/03/dapr-distributed-application-runtime-joins-cncf-incubator/) in November 2021. The [proposal](https://github.com/cncf/toc/blob/main/proposals/incubation/dapr.md) and [due diligence](https://docs.google.com/document/d/1GknuJk8Z26KhvFJiblbMSopWbX27nihpxkEG0_TpkCw/) are provided here for reference.
+
+## Graduation criteria requirements
+
+Dapr meets the graduation stage requirements in the following ways:
+
+### Have committers from at least two organizations.
+
+Dapr makes use of the term "[maintainer](https://github.com/dapr/community/blob/master/community-membership.md#maintainer)" as the noun for a member allowed to accept and merge code contributions. To date, the project has [14 active maintainers](https://github.com/dapr/community/blob/master/MAINTAINERS.md), representing 4 organizations.
```
[the remark this answers is not in the document]
Comment (yaron2) [reply proposals/graduation/dapr.md:13]: That is correct.

**Comment by dims** [comment]
Dear Dapr folks, we (the TOC) have reworked the process(es) a bit since this PR was filed, Can you please review the latest in:
https://github.com/cncf/toc/blob/main/process/README.md#applying-to-become-an-incubating-or-graduating-project

There's a slightly longer form/issue to fill in:
https://github.com/cncf/toc/blob/main/process/README.md#applications-to-move-levels-are-done-by-submitting-an-incubation-or-graduation-application-issue-on-the-toc-repo

I will hopefully be your TOC sponsor as soon as the new issue is filed. we may have another TOC member join me as well for this process.

**Comment by dims** [comment]
cc @yaron2 @msfussell @daixiang0 @jackzhao @olitomlinson @johnewart @paulyuk

( current steering committee members listed here - https://github.com/dapr/community/blob/master/steering-and-technical-committee-charter.md#committee-members )

**Comment by yaron2** [comment]
> Dear Dapr folks, we (the TOC) have reworked the process(es) a bit since this PR was filed, Can you please review the latest in: https://github.com/cncf/toc/blob/main/process/README.md#applying-to-become-an-incubating-or-graduating-project
> 
> There's a slightly longer form/issue to fill in: https://github.com/cncf/toc/blob/main/process/README.md#applications-to-move-levels-are-done-by-submitting-an-incubation-or-graduation-application-issue-on-the-toc-repo
> 
> I will hopefully be your TOC sponsor as soon as the new issue is filed. we may have another TOC member join me as well for this process.

Great to see progress on this! @msfussell will fill the form today.

**Comment by daixiang0** [comment]
Great to see it!

**Comment by angellk** [comment]
Closing and will continue on #1354 - thanks @msfussell !

**Pull request closed without merging**